data/pc/1.19.2/protocol.json

```diff
@@ -3195,13 +3208,77 @@
               "type": "i64"
             },
             {
-              "name": "signature",
+              "name": "previousMessages",
               "type": [
-                "buffer",
+                "array",
                 {
-                  "countType": "varint"
+                  "countType": "varint",
+                  "type": [
+                    "container",
+                    [
+                      {
+                        "name": "previousMessageSender",
```

Comment:
this can just be "messageSender" and "messageSignature" as previous is implied

Reply:
done